src/container/container.rs

```diff
@@ -36,10 +39,12 @@ impl Container {
     pub fn status(&self) -> ContainerStatus {
         self.state.status
     }
-
     pub fn refresh_status(&self) -> Result<Self> {
         let new_status = match self.pid() {
             Some(pid) => {
+                // Note that Process::new does not spawn a new process
+                // but instead creates a new Process structure, and fill
+                // it with information about the process with given pid
```

Comment:
This is a good comment. I, the author, didn't think anything of it, but it certainly could be misunderstood.

Reply:
Yes, the first time I was looking at this while commenting create.rs, I was wondering why we create a new process here, then I checked the docs of process and found out it fills new structure. So I kept that in mind when writing this :)